wweval/R/ms_fig4.R

```diff
@@ -297,21 +432,27 @@ make_fig4_rel_crps_overall <- function(scores,
     )
 
   relative_crps <- scores_comb |>
-    compute_relative_crps(id_cols = c(
-      "location", "forecast_date",
-      "horizon", "date"
-    )) |>
-    dplyr::filter(!is.na(horizon)) |>
+    dplyr::group_by(forecast_date, location, model, horizon) |>
+    dplyr::summarize(mean_crps = mean(crps)) |>
+    tidyr::pivot_wider(
+      names_from = model,
+      values_from = mean_crps,
+      id_cols = c("horizon", "forecast_date", "location")
+    ) |>
+    dplyr::mutate(
+      rel_crps = ww / hosp
+    ) |>
```

Comment:
Not for this PR, but great candidate for DRY as it comes up a lot.

Reply:
Yeah I ended up having to recompute each one since we no longer use  the function we originally wrote (`compute_rel_crps()`). I find it easier to just see how we summarized each time within the figure function, since it is not identical in all cases.

Reply:
Created #213